src/Server.php

```diff
@@ -6,6 +6,10 @@
 use React\Socket\ServerInterface as SocketServerInterface;
 use React\Socket\ConnectionInterface;
 use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use RingCentral;
+use React\Stream\ReadableStream;
+use React\Promise\Promise;
```

Comment:
Since you're starting to use `react/promise` explicitely, i recommend adding it to the composer.json. At the moment `react/promise` is only indirectly required via `react/socket`.